app/controllers/api/v0/users/games_controller.rb

```diff
@@ -55,6 +55,59 @@ def current_round
     end
   end
 
+  def vote
+    begin
+      user = User.find(params[:user_id])
+      game = Game.find(params[:game_id])
+      round = game.current_round
+      user_game = UserGame.find_by(user_id: user.id, game_id: game.id)
+    rescue ActiveRecord::RecordNotFound => exception
+      render json: ErrorSerializer.new(ErrorMessage.new(exception.message, 404)).error_json, status: :not_found
+    end
+    if user_game&.accepted? || user_game&.admin?
+      if round.open?
+        vote = Vote.find_by(user_id: params[:user_id],
+                            round_id: round.id)
+        if vote
+          vote.update(lat: params[:lat], 
+                      lon: params[:lon], 
+                      user_id: params[:user_id],
+                      round_id: round.id)
+          render json: VoteSerializer.new(vote), status: :ok
+        else
+          vote = Vote.create!(lat: params[:lat], 
+                              lon: params[:lon], 
+                              user_id: params[:user_id],
+                              round_id: round.id)
+          render json: VoteSerializer.new(vote), status: :created
```

Comment:
The logic here looks great to support with how to handle an Updated Vote and Creating a new Vote.